### PR TITLE
Added a reply

### PR DESCRIPTION
**Pull request by scoopandrun**
We need a little more conversation in this world 😁

### CODE REVIEW

**Comment by verv0eren** [comment]
I see you have forked well, man! 😜

File: index.html [hunk 1]
```diff
@@ -1,20 +1,26 @@
 <!DOCTYPE html>
 
 <html>
+
 <head>
-  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
+  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
```
Comment (verv0eren) [inline index.html:6]: This was not needed 😂

**Review by verv0eren — COMMENTED** [review]
Just tweaking things around to see how all this works.

File: index.html [hunk 2]
```diff
@@ -1,20 +1,26 @@
...
+  <p>
+    Fork me? Fork you, @octocat!
+  </p>
 
-<!-- Feel free to change this text here -->
-<p>
-  Fork me? Fork you, @octocat!
-</p>
+  <p>
+    You too, junk!
```
Comment (verv0eren) [inline index.html:21]: 😮

File: index.html (hunk 1 above)
Comment (scoopandrun) [reply index.html:6]: Editor auto-formatting 😉

**Pull request closed without merging**